com.unity.render-pipelines.universal/Runtime/UniversalRendererData.cs

```diff
@@ -78,8 +78,6 @@ public sealed class ShaderResources
 
         public ShaderResources shaders = null;
 
-        const int k_LatestAssetVersion = 1;
-        [SerializeField] int m_AssetVersion = 0;
```

Comment:
Should this be removed?

Reply:
Yep, since there's no code on 2021.2 making use of the upgrade path.